src/style.css

```diff
@@ -1,3 +1,71 @@
+:root {
+  --letters-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
Ensure that the font 'Roboto' is correctly loaded in your HTML file. If it's not available, the browser will fall back to 'sans-serif'.